assets/templates/confirmation-page.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Doppler UI Team</title>
+  <meta name="keywords" content="">
+  <meta name="description" content="Confirmation page">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="canonical" href="" />
+  <!-- Open Graph metatags -->
+  <meta property="og:title" content="Confirmation page" />
+  <meta property="og:type" content="article" />
+  <meta property="og:image" content="Confirmation page" />
+  <meta property="og:url" content="Confirmation page" />
+  <meta property="og:description" content="Confirmation page" />
+  <meta property="og:site_name" content="Confirmation page" />
+  <!-- Stylesheets and polyfills -->
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <!-- Favicons -->
+  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.fromdoppler.com/apple-touch-icon-144x144-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.fromdoppler.com/apple-touch-icon-114x114-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.fromdoppler.com/apple-touch-icon-72x72-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="57x57" href="https://www.fromdoppler.com/apple-touch-icon-57x57-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" href="https://www.fromdoppler.com/apple-touch-icon-precomposed.png" />
+  <link rel="apple-touch-icon" href="https://www.fromdoppler.com/apple-touch-icon.png" />
+  <link rel="icon" type="image/x-icon" href="https://www.fromdoppler.com/favicon.ico" />
+  <link rel="shortcut icon" type="image/x-icon" href="https://www.fromdoppler.com/favicon.ico" />
+</head>
+
+<body>
+  <div class="confirmation-wrapper">
+    <header class="confirmation-header">
+      <h1 class="logo-doppler-new">Doppler</h1>
+    </header>
+    <main class="confirmation-main">
+      <article class="confirmation-article">
+        <h2>Gracias por registrarte</h2>
+        <p>Revisa tu casilla ¡Tienes un email!</p>
+        <span class="icon-registration m-bottom--lv6">Fijate en tu correo electronico</span>
```

Comment:
If this is in neutral spanish should be `Fíjate`. Also missing accent: `electrónico`

Reply:
The content is no final @cbernat

Reply:
No importa si no es final, estaría bueno que esté lo más correcto posible. Por favor, aplicá las sugerencias de Ceci.